web/assets/theme/colors.css

```diff
@@ -0,0 +1,65 @@
+/* CSS - Cascading Style Sheet */
```

Comment:
bootstrap enables css sass processor.  Keeping this file for future change, where Sass is used to create `$` variables to use in `gerbera.css`.  Eventually make the gerbera CSS responsive to bootstrap compiled sass --> css.